drivers/AnalogIn.h

```diff
@@ -93,6 +93,35 @@ class AnalogIn {
      */
     unsigned short read_u16();
 
+    /**
+     * Read the input voltage in volts. The output depends on the target board's
+     * ADC reference voltage (typically equal to supply voltage). The ADC reference voltage
+     * sets the maximum voltage the ADC can quantify (ie: Vin == Vref when ADC output == ADC_MAX_VALUE)
+     *
+     * The target's ADC reference voltage can be configured by overriding "drivers.adc_vref"
+     *
+     * @returns A floating-point value representing the current input voltage, measured in volts.
+     */
+    float read_volts();
```

Comment:
As we `set_reference_voltage` and `get_reference_voltage`, it feels more natural to me to also `read_voltage` to keep the use of the word `voltage`.

Reply:
I agree, the wording should be more consistent.

Updated, see f965096